frontend/src/app/AppRoutes.tsx

```diff
@@ -70,8 +66,6 @@ const AppRoutes: React.FC = () => {
         <Route path="/pipelines/*" element={<GlobalPipelinesRoutes />} />
         <Route path="/pipelineRuns/*" element={<GlobalPipelineRunsRoutes />} />
 
-        <Route path="/dependency-missing/:area" element={<DependencyMissingPage />} />
```

Comment:
I'm on the fence, but I think it would be great to keep this here -- I know it's kinda goofy, but I think it has low harm too... and showcases how to do it in the future if we need it.

Reply:
brought it back